### PR TITLE
--feat：贪吃蛇 

### PR DESCRIPTION
**Pull request by Cuphoria**
添加code stream 体验

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: codestream/snake.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	width:25,
+	height:25
+};
+//格子个数
+var boxNums={
+	wNums:map.width/box.width,
+	hNums:map.height/box.height,
+	nums:0
+};
+boxNums.nums=boxNums.wNums*boxNums.hNums;
```
Comment (Cuphoria) [inline codestream/snake.js:16]: 写死了还要算:sweat_smile:

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 纸上得来终觉浅，绝知此事要躬行
```
Comment (Cuphoria) [inline README.md:1]: good

**Review by Cuphoria — COMMENTED** [review]
self review done

**Comment by Cuphoria** [comment]
@pengkunbin 评论下

File: codestream/snake.js (hunk 1 above)
Comment (pengkunbin) [reply codestream/snake.js:16]: :rolling_on_the_floor_laughing:

File: README.md (hunk 2 above)
Comment (pengkunbin) [reply README.md:1]: :exploding_head:

**Review by pengkunbin — APPROVED** [review]
LGTM

**Review by pengkunbin — APPROVED** [review]

**Pull request merged**